template/Cargo.toml

```diff
@@ -4,7 +4,5 @@ version = "0.1.0"
 edition = "2018"
 
 [dependencies]
-# In a real application, change these to point to the latest release
-# on crates.io.
-uefi = { path = "..", features = ["exts"] }
-uefi-services = { path = "../uefi-services" }
+uefi = { version = "0.13.0", features = ["exts"] }
+uefi-services = "0.10.0"
```

Comment:
Could you perhaps document in `PUBLISHING.md` that we ought to update these versions here as well when we update the main crate version? I'm thinking we wouldn't want the template to fall too much behind the published versions.

Reply:
Added a line to that effect. Also updated these deps to the latest version.